### ISSUE TITLE
Empty lines are lost when copy-pasting from code section.

### ISSUE THREAD

**Issue by Knitschi** [doxygen/doxygen#6812]
When copy pasting from code sections that are generated with the `\code ... \endcode` commands, empty lines are lost. 

I have a part in my documentation, where I present the content of some files in a \code section. When you try to follow the tutorial by copy-pasting the content of the files, the empty lines are lost which makes the copied code harder to read.

This can be reproduced by opening the doxygen documentation

http://www.doxygen.nl/manual/commands.html#cmdsnippet

and copy the content of the section

```
    QImage image(64, 64, QImage::Format_RGB32);
    image.fill(qRgb(255, 160, 128));

//! [Adding a resource]
    document->addResource(QTextDocument::ImageResource,
        QUrl("mydata://image.png"), QVariant(image));
//! [Adding a resource]
...
```

which contains an empty line. Paste to a text editor and that line is lost.

As a workaround one can use `\verbatim ... \endverbatim` which does not suffer from that problem.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #6813

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
